ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/presentation.tsx

```diff
@@ -70,6 +77,54 @@ const ResultsExport = (props: Props) => {
           renderInput={(params) => <TextField {...params} variant="outlined" />}
         />
       </div>
+      {['CSV', 'RTF', 'XLSX'].includes(selectedFormat) ? (
```

Comment:
This `Manage Attributes` button looks the same as the one in [.../table-export/presentation.tsx](https://github.com/codice/ddf-ui/pull/818/files#diff-9e6f57d0c85f4b0dd6598845c56484e24289a85b568f19e2884da7bbf983be0fR125). Maybe we could have it be its own separate component?